nova/templates/hook-scripts/postStart-nova-compute.py

```diff
@@ -0,0 +1,67 @@
+#!/bin/python
+import os
+import socket
+import yaml
+
+from keystoneauth1 import identity, session
+from novaclient import client, exceptions
+
+OPENRC_PATH = "/test.yaml"
```

Comment:
We actually have an openrc shell script exporting the required env variables: `nova/templates/configmaps/openrc.yaml.j2`.